.github/workflows/update.yml

```diff
@@ -2,7 +2,7 @@ name: Automatic Updates
 
 on:
   schedule:
-    - cron: 0 0 * * *
+    - cron: 0 0 * * 1,4
```

Comment:
Why twice? I think that once is often enough.

Reply:
```suggestion
    - cron: 0 0 * * 1
```

Reply:
Fine either way. Let's start with once a week then!